components/enterprise-contract/rekor-local/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base/
+  - rekor.yaml
+
+patches:
+  - tekton-chains-config.yaml
+#  - remove-rekor-ingress.yaml
```

Comment:
Is this line intentionally commented out?

Reply:
Doh, forgot to uncomment that one

Reply:
Did it unnecessarily complicated before, now it should be okay